src/test/java/org/openmaptiles/layers/TransportationTest.java

```diff
@@ -1176,7 +1809,7 @@ void testFerry() {
       "_layer", "transportation",
       "class", "ferry",
 
-      "_minzoom", 11,
+      "_minzoom", 5,
       "_maxzoom", 14,
```

Comment:
This test seems to be failing, I think we just need to change to this to capture the minPixelSize change:

```suggestion
      "_minzoom", 4,
      "_maxzoom", 14,
      "_minpixelsize", 32d,
```

Reply:
Thank you, adjusted.